addons/mass_mailing/views/snippets/s_blockquote.xml

```diff
@@ -12,7 +12,7 @@
                     <div>
                         <p><i>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</i></p>
                         <div>
-                            <img src="/mass_mailing/static/src/img/theme_default/team_member_2-circle.png" style="width: 40px" class="img me-2" data-original-id="361" data-original-src="/mass_mailing/static/src/img/theme_default/s_default_image_team_member_2.jpg" data-mimetype="image/svg+xml" data-shape="mass_mailing/basic/circle" data-file-name="team_member_2-circle.png" data-shape-colors=";;;;" data-original-mimetype="image/png"/>
+                            <img src="/web/image/mass_mailing.s_company_team_default_image_2-circle" style="width: 40px" class="img me-2" data-original-id="361" data-original-src="/mass_mailing/static/src/img/theme_default/s_default_image_team_member_2.png" data-mimetype="image/svg+xml" data-shape="mass_mailing/basic/circle" data-file-name="s_default_image_team_member_2-circle.png" data-shape-colors=";;;;" data-original-mimetype="image/png"/>
```

Comment:
I think this is fixed in https://github.com/odoo/odoo/pull/112262. Could you please try reverting this commit here and cherry-picking the ones from that PR to confirm ?

Reply:
Yep the pr fix the issue.